src/node.cpp

```diff
@@ -1382,7 +1368,7 @@ void LocalNode::init(Sync* csync, nodetype_t ctype, LocalNode* cparent, string*
     // enable folder notification
     if (type == FOLDERNODE)
     {
-        sync->dirnotify->addnotify(this, cfullpath);
```

Comment:
Yes this is another case where the PR would be ok with fewer lines changed - in fact, for windows we ignore that parameter anyway, and for the other platforms it's utf8 in either case.    But we certainly could convert addnotify() in future, it would make good sense.